storage/innobase/xtrabackup/src/backup_copy.cc

```diff
@@ -1443,6 +1443,9 @@ bool backup_start(Backup_context &context) {
                  false);
 
   context.log_status = log_status_get(mysql_connection);
+  os_thread_sleep(10000000);
```

Comment:
Did you left this by mistake?

Reply:
no , I was doing some testing. Should have created another branch to test. removing it.